docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -11,9 +11,7 @@ title: StatefulSets
 
 {% capture overview %}
 **StatefulSets are a beta feature in 1.7. This feature replaces the
```

Comment:
I would remove the reference to PetSets at this point. It is not possible to upgrade to 1.8 without going through 1.7 and we have a reference to it in those docs. Perhaps 
"StatefulSet is the workload API object used to manage stateful applications. StatefulSets are beta in 1.8."

Reply:
Sure, I will remove all references to PetSets.